### ISSUE TITLE
[TC-PRS-2.1] MaxMeasuredValue range has to be updated as per Spec

### ISSUE THREAD

**Issue by Ashwinigrl** [project-chip/matter-test-scripts#157]
### Description:

- [ ] TC-PRS-2.1

   **Step 3** : As per spec MaxMeasuredValue value range is from MinMeasuredValue+1 to 32767, but script is verifying range from MinMeasuredValue to 32767. So script has to be updated with valid range of MaxMeasuredValue .

### Steps to reproduce:
Step 3

### Spec References:
![image](https://github.com/project-chip/matter-test-scripts/assets/98016634/6d43f59b-ff17-451c-b75c-34b6f456903e)

 - https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/PressureMeasurement.adoc#5-attributes
 
### Yaml References:
![image](https://github.com/project-chip/matter-test-scripts/assets/98016634/7f7eb723-201a-4cfd-8260-b2d438cf43d9)

 - https://github.com/project-chip/connectedhomeip/blob/master/src/app/tests/suites/certification/Test_TC_PRS_2_1.yaml

**Comment by raul-marquez-csa** [comment]
Addressed in PR
https://github.com/project-chip/connectedhomeip/pull/32296

**Comment by Ashwinigrl** [comment]
Issue is not fixed in PR https://github.com/project-chip/connectedhomeip/pull/32296

**Comment by raul-marquez-csa** [comment]
@Ashwinigrl, can you please verify if issue persists?

**Comment by Ashwinigrl** [comment]
Yes @raul-marquez-csa, In Test Step 3, the expected range for MaxMeasuredValue is from (MinMeasuredValue + 1) to 32767. But, the script is incorrectly verifying the range from MinMeasuredValue to 32767.

**Spec Ref:**

![image](https://github.com/user-attachments/assets/d68208df-08a2-4bc9-89d5-8fd37c039003)

**Script Ref:**

![image](https://github.com/user-attachments/assets/f08f9d5c-7b77-47b4-9588-0ed932fa8441)

**Comment by raul-marquez-csa** [comment]
PR
[TC-PRS-2.1 - Updates minValue in step 3 as per VS #36311
](https://github.com/project-chip/connectedhomeip/pull/36311)
